zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterGroup.java

```diff
@@ -119,4 +120,27 @@ public void run() {
       }
     }
   }
+
+  public void bringDefaultToFront() {
+    String defaultInterpreterName = getProperty().getProperty("zeppelin.default.interpreter");
+    Logger logger = Logger.getLogger(InterpreterGroup.class);
+    logger.info("Default interpreter name is " + defaultInterpreterName);
+    if (defaultInterpreterName != null && defaultInterpreterName.trim().length() > 0) {
+      Interpreter defaultInterpreter = null;
+      for (Interpreter intp : this) {
+        RegisteredInterpreter regIntp = Interpreter
+            .findRegisteredInterpreterByClassName(intp.getClassName());
+        if (regIntp.getName().equals(defaultInterpreterName)) {
```

Comment:
Could you override/create the LinkedList<Interpreter> contains method with the above? I'm sure people will want the capacity to search the group by interpreter name.